src/cart/crt_init.c

```diff
@@ -348,6 +348,9 @@ static int data_init(int server, crt_init_options_t *opt)
 	}
 	if (server)
 		d_setenv("UCX_IB_FORK_INIT", "n", 1);
+	else
+		d_setenv("UCX_IB_FORK_INIT", "y", 1);
```

Comment:
Now thinking about it a bit more, we should only set this to 'y' conditionally if it is not explicitly set to 'n' by the client already (similar to how we handle universe size setting few lines below). 

Some clients that know that they don't do forks might want to run with a explicit 'n' setting to get a slightly more optimized/faster startup.